README.md

```diff
@@ -20,6 +20,7 @@ maintenance. To non-destructively update your development stack at any time:
 1. Run `docker-compose pull`
 2. Run `docker-compose build --pull --no-cache`
 3. Run `docker-compose run --rm django ./manage.py migrate`
+4. Run `docker-compose run --rm arangodb arangod --database.auto-upgrade`
```

Comment:
What does this command do?

Reply:
It upgrades the `_system` database. Here's a link to the [docs](https://www.arangodb.com/docs/stable/programs-arangod-database.html#auto-upgrade)

Relevant lines:
> ... the server will check whether there are any upgrade tasks to perform. It will then execute all required upgrade tasks and print their statuses ...

> Whether or not this option is specified, the server will always perform a version check on startup. Running the server with a non-matching version number in the VERSION file will make the server refuse to start.

Thus it must be ran whenever the minor version changes